### PR TITLE
Miscellaneous rsync fixes

### PR DESCRIPTION
**Pull request by csrwng**
Miscellaneous fixes for rsync in windows and tar strategy:
- Added verbose logging of tar extract when extracting to local machine
- Fixed truncation of directory name when archiving a directory relative to current directory (tar)
- Rsync in windows not falling back to tar when rsync not present in container
- Added check for presence of tar in remote container if tarring failed
- Added descriptions for each strategy

### CODE REVIEW

**Comment by csrwng** [comment]
@bparees or @smarterclayton ptal

File: pkg/cmd/cli/cmd/rsync/rsync.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ for the copy.`
   $ %[1]s POD:/remote/dir/ ./local/dir`
 
 	noRsyncUnixWarning    = "rsync command not found in path. Please use your package manager to install it."
-	noRsyncWindowsWarning = "rsync command not found in path. Download cwRsync for windows and add it to your path."
+	noRsyncWindowsWarning = "rsync command not found in path. Download cwRsync for Windows and add it to your path."
```
Comment (smarterclayton) [inline pkg/cmd/cli/cmd/rsync/rsync.go]: to your PATH

Comment (csrwng) [inline pkg/cmd/cli/cmd/rsync/rsync.go]: updated

**Comment by csrwng** [comment]
[testonlyextended][extended:core(rsync)]

**Comment by smarterclayton** [comment]
LGTM [merge]

**Comment by csrwng** [comment]
integration test flake [merge] again

**Comment by csrwng** [comment]
[merge]

**Pull request merged**